tutorials/settings_tutorial.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Settings Tutorial
+---
+<!-- Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+## Introduction
+As discussed in the _Settings_ Guide page, plugins can add Settings to IntelliJ Platform-based IDEs. 
+The IDE displays the Settings of plugins in response to a user choosing **Settings/Preferences**. 
+Custom Settings are displayed and function just like those native to the IDE.
+
+* bullet list
+{:toc}
+
+## Overview of a Custom Settings Implementation
+Using the SDK code sample [`settings`](https://github.com/JetBrains/intellij-sdk-docs/tree/master/code_samples/settings), this tutorial illustrates the steps to create custom Application-level Settings. 
+As the diagram below shows, three classes are used to implement custom Application Settings: [`AppSettingsConfigurable`](https://github.com/JetBrains/intellij-sdk-docs/tree/master/code_samples/settings/src/main/java/org/intellij/sdk/settings/AppSettingsConfigurable.java), [`AppSettingsState`](https://github.com/JetBrains/intellij-sdk-docs/tree/master/code_samples/settings/src/main/java/org/intellij/sdk/settings/AppSettingsState.java), and [`AppSettingsComponent`](https://github.com/JetBrains/intellij-sdk-docs/tree/master/code_samples/settings/src/main/java/org/intellij/sdk/settings/AppSettingsComponent.java). 
+
+ !["Settings Class Diagram"](img/settings_class.png){:width="800px"}
```

Comment:
not sure we need this, if yes, we must setup/align presentation vs the diagrams we have in IJ